config/config.linux

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+echo " Defaults for Linux machine"
+
+# set CUQUANTUM dir
+CUQUANTUM_DIR=~/qs/cuquantum-linux-x86_64-22.11.0.13-archive
+
+# set CUDA dir in case cmake cannot autodetect a path
+CUDA_DIR=/usr/local/cuda-11.7
```

Comment:
Is this part specific to your machine? Is there a way to make them environment variables so that we don't need to push this part to the codebase?

Reply:
I tried to use this cmake script to set all the related lib path and decide whether to build the simulator. If QUARTZ_USE_SIMULATOR is set to OFF,  it will only build quartz_runtime. When it is set to ON, user can set the above path for cmake to know the cuda-related path. We can also do this manually in terminal by setting the env variables and this is just for convenience when there are many things to be set.